webapp/components/customTunnelsThroughPartner/stargate.tsx

```diff
@@ -0,0 +1,25 @@
+import Image from 'next/image'
+import { useTranslations } from 'next-intl'
+
+import { PartnerLink } from './partnerLink'
+import stargateLogo from './partnerLogos/stargate.svg'
+
+export const Stargate = function () {
+  const t = useTranslations('tunnel-page.tunnel-partners')
+  return (
+    <PartnerLink
+      icon={
+        <Image
+          alt="Stargate Banner"
+          className="mr-2 rounded-lg bg-black p-1.5"
+          height={32}
+          src={stargateLogo}
+          width={32}
+        />
+      }
+      text={t('tunnel-with-stargate')}
+      // URL is TBD, see https://github.com/hemilabs/ui-monorepo/issues/719
```

Comment:
Let's use the `// TODO xxx` format for future comments:

```suggestion
      // TODO URL is TBD, see https://github.com/hemilabs/ui-monorepo/issues/719
```

There is a nice VSC extension called "Todo tree" to list all the TODOs and FIXMEs in a repo.